templates/tomcat/logging.properties

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I am not yet clear on if we need to carry thi smodified file, I was playing around with whether pure console logging would give more information for debugging through the container based deployment.